cloud/scope/build.go

```diff
@@ -602,3 +611,24 @@ func (s *BuildScope) PatchObject() error {
 func (s *BuildScope) Close() error {
 	return s.PatchObject()
 }
+
+func (s *BuildScope) ImageName() string {
+	return fmt.Sprintf("%s-%s", "forge", s.Name())
+}
+
+func (s *BuildScope) IsProvisionerReady() bool {
+	return s.Build.Status.ProvisionersReady
+}
+
+func (s *BuildScope) IsReady() bool {
+	return s.GCPBuild.Status.Ready
+}
+
+func (s *BuildScope) IsCleanedUp() bool {
+	return s.GCPBuild.Status.CleanUpReady
+}
+
+// Implement the method to return the Compute service
+func (b *BuildScope) GetComputeService() *compute.Service {
```

Comment:
Why you need this here?

Reply:
The GetComputeService() function will be used to initialize the `InstanceService()`.
> Why do we use `InstanceService()` Instead of `Cloud().Instances()`? 

Because InstanceService() implement functions like `Stop()`, `Start()`, `AttachDisk()`, while `Cloud().Instances()` can only `Get()`, `Insert()`, `Delete()`, `List()`